pkg/cmd/server/api/types.go

```diff
@@ -325,6 +325,9 @@ type MasterNetworkConfig struct {
 	ClusterNetworkCIDR string
 	HostSubnetLength   uint
 	ServiceNetworkCIDR string
+	// ExternalIPNetworkCIDR controls what values are acceptable for the service external IP field. If empty, no externalIP
```

Comment:
That's a strange way to deal with empty string and it makes it impossible to default to existing behavior, doesn't it?

If you moved this to the admission config section, then no config means "allow all" (existing behavior) and config with empty string can mean "restrict all".  That seems to be more intuitive.